src/database/sql_format.h

```diff
@@ -0,0 +1,60 @@
+/*GRB*
+  Gerbera - https://gerbera.io/
+
+  sql_format.h - this file is part of Gerbera.
+
+  Copyright (C) 2018-2021 Gerbera Contributors
+
+  Gerbera is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License version 2
+  as published by the Free Software Foundation.
+
+  Gerbera is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with Gerbera.  If not, see <http://www.gnu.org/licenses/>.
+
+  $Id$
+*/
+
+/// \file sql_format.h
+
+#ifndef __SQL_FORMAT_H__
+#define __SQL_FORMAT_H__
+
+#include <fmt/format.h>
```

Comment:
I think this should be fmt/core.h

Reply:
Fix applied locally, will push when there are more changes. Thanks for pointing it out.